Adapter_Revit_UI/CRUD/Update.cs

```diff
@@ -48,12 +49,68 @@ public partial class RevitUIAdapter
         //    if (bhomObject is IView || bhomObject is oM.Adapters.Revit.Elements.Family || bhomObject is InstanceProperties)
         //        element.Name = bhomObject.Name;
 
+        //FOR REFERENCE, IF OBJECT IS REVITFILEPREVIEW:
+        //TODO: this value should come from adapter PushType?
+        //bool updateFamilies = true;
+        //FamilyLoadOptions familyLoadOptions = new FamilyLoadOptions(updateFamilies);
+        //if (document.LoadFamily(revitFilePreview.Path, out family))
+        //{
+        //    SetIdentifiers(bhomObject, family);
+        //    element = family;
+        //}
+        //}
+
         /***************************************************/
         /****              UpdateProperty               ****/
         /***************************************************/
 
         // This method used to be called from the UpdateProperty component
         // Now it is never called as it doesn't override anymore
         // This logic should be called from the Push component instead
```

Comment:
Remove, I'd say?

Reply:
To be done in the next PR which will focus on update.